.travis.yml

```diff
@@ -3,5 +3,6 @@ git:
   depth: 2
 language: node_js
 node_js:
+  - 'iojs'
```

Comment:
Why is that? I see people using quotes everywhere

http://docs.travis-ci.com/user/languages/javascript-with-nodejs/

https://github.com/gruntjs/grunt-contrib-connect/blob/master/.travis.yml